MAINTENANCE.md

```diff
@@ -94,12 +94,18 @@ There are a few steps that still must be done manually, though.
    to make the change automatically.
 4. Copy the remaining changes to `doc/changes.rst` under a new section for the
    intended release.
-5. Reserve a DOI in [Zenodo](https://zenodo.org/) and pre-fill the fields for archiving
-   the new release. Include as authors anyone who made contributions between now and the
-   last release.
-6. Include the DOI badge in the changelog.
-5. Add a link to the new release version documentation in `README.rst`.
-5. Open a new PR with the updated changelog.
+5. Add a list of people who contributed to the release (use
+   `` git shortlog HEAD...v0.1.2 -sne ``).
+6. Include the DOI badge in the changelog. Remember to replace your DOI inside the badge url.
+
+    ```
+    .. image:: https://zenodo.org/badge/DOI/<INSERT-DOI-HERE>.svg
+        :alt: Digital Object Identifier for the Zenodo archive
+        :target: https://doi.org/<INSERT-DOI-HERE>
+    ```
```

Comment:
@seisman if you have time, do you mind reserving a [Zenodo](https://zenodo.org) DOI for pygmt? It's not showing up on my Zenodo account for some reason.

Reply:
I also don't see it on my Zenodo account.

Reply:
Might need to add the zenodo oauth app https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/approving-oauth-apps-for-your-organization

Reply:
I'm done with the oauth app. Perhaps you can check it shows up in your account, since I have limited knowledge about zenodo.

Reply:
Thanks! I can see it in my account now, will try to figure it out (I've only used it once too).